tools/explorer/pkg/escrow/stellar.go

```diff
@@ -482,11 +491,11 @@ func (e *Stellar) createOrLoadAccount(customerTID int64) (string, error) {
 	return res.Address, nil
 }
 
-func addressByAsset(addrs []gdirectory.WalletAddress, asset string) (string, error) {
+func getAddressFarmer(addrs []gdirectory.WalletAddress) (string, error) {
```

Comment:
I don't think you should hardcode the asset in here. Let's keep this function generic.